climada/util/plot.py

```diff
@@ -472,3 +472,70 @@ def get_transformation(crs_in):
     except KeyError:
         units = '°'
     return crs_epsg, units
+
+
+def bar_plot(ax, data, colors=None, total_width=0.8, single_width=1, legend=True):
```

Comment:
perhaps rather name the function `multibar` or `multibar_plot`? So that it's obviously an addition to the normal `plt.bar()`